addons/mail/models/models.py

```diff
@@ -142,7 +145,7 @@ def _notify_get_reply_to(self, default=None, records=None, company=None, doc_nam
                 result[res_id] = self._notify_get_reply_to_formatted_email(
                     result_email[res_id],
                     doc_names.get(res_id) or '',
-                    company or self.env.company
+                    company
```

Comment:
Note that the parameter was completely removed in newer versions ... not sure, if that's just for a company name that is anyway removed if > 78 chars, should we fix it ?

Reply:
I just removed it here because it is redundant (no change in behaviour)
the default value  self.env.company is still applied inside  `_notify_get_reply_to_formatted_email()`

Reply:
OK then :)